### PR TITLE
chore(04-channel): be consistent when formatting enums

### PR DESCRIPTION
**Pull request by tropicaldog**
## Description

closes: #3946 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against the correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to GitHub issue with discussion and accepted design, OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/build/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [conventional commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to follow the repository standards.
- [ ] Include a descriptive changelog entry when appropriate. This may be left to the discretion of the PR reviewers. (e.g. chores should be omitted from changelog)
- [ ] Re-reviewed `Files changed` in the GitHub PR explorer.
- [ ] Review `SonarCloud Report` in the comment section below once CI passes.

## Summary by CodeRabbit

- **Bug Fixes**
	- Improved clarity and consistency of error messages across various channel operations.
	- Enhanced error handling in timeout operations for better stability.

### CODE REVIEW

**Review by crodriguezvega — APPROVED** [review]
Looks good to me. Thank you, @tropicaldog!

File: modules/core/04-channel/keeper/timeout.go [hunk 1]
```diff
@@ -112,7 +112,7 @@ func (k *Keeper) TimeoutPacket(
 			packet.GetDestPort(), packet.GetDestChannel(), packet.GetSequence(),
 		)
 	default:
-		panic(errorsmod.Wrapf(types.ErrInvalidChannelOrdering, channel.Ordering.String()))
+		panic(errorsmod.Wrap(types.ErrInvalidChannelOrdering, channel.Ordering.String()))
```
[the remark this answers is not in the document]
Comment (tropicaldog) [reply modules/core/04-channel/keeper/timeout.go:115]: > panic(errorsmod.Wrap(types.ErrInvalidChannelOrdering, channel.Ordering))

this is wrong, errorsmod.Wrap accept string as the second argument, where channel.Ordering isn't a string. 

Could be refactor to `panic(errorsmod.Wrapf(types.ErrInvalidChannelOrdering, "%s", channel.Ordering))` in order to remove `.String()` calls, but I'm not sure if it is desirable @DimitrisJim

[the remark this answers is not in the document]
Comment (crodriguezvega) [reply modules/core/04-channel/keeper/timeout.go:115]: I think the only other places where we could potentially remove the `.String()` is in the log statements. Is it ok if we just pass the enum value in those?

[the remark this answers is not in the document]
Comment (tropicaldog) [reply modules/core/04-channel/keeper/timeout.go:115]: > can we apply that change?

Yeah my bad I forgot to push the changes. Also thanks @crodriguezvega for the help!

[the remark this answers is not in the document]
Comment (crodriguezvega) [reply modules/core/04-channel/keeper/timeout.go:115]: No problem, @tropicaldog. Thanks for opening the PR!

**Comment by tropicaldog** [comment]
> ayo @tropicaldog can you merge main? For some reason I cannot

Done sir, please check it out.

**Comment by crodriguezvega** [comment]
> thanks! will merge after one more review that is required

@DimitrisJim If that's ok with you, I am going to merge it now.

**Pull request merged**